app/assets/javascripts/application.js.coffee

```diff
@@ -173,6 +173,16 @@ class Homs
       cache: true
       language: I18n.locale)
 
+  onUpdateForm: () =>
```

Comment:
method name does not require it to be a callback, so we can go with `updateForm` or sutin